services/github/scopes.yml

```diff
@@ -3,3 +3,4 @@
 - queue:route:statuses
 - queue:route:checks
 - assume:scheduler-id:taskcluster-github/*
+- assume:anonymous
```

Comment:
Nit: git will complain without a newline at the end of the file, here and other places.